src/sentry/relay/globalconfig.py

```diff
@@ -14,7 +14,8 @@
     "relay.span-usage-metric",
     "relay.cardinality-limiter.mode",
     "relay.cardinality-limiter.error-sample-rate",
-    "relay.metric-bucket-encodings",
```

Comment:
Will it create compatibility issues if we rename this? Should `relay.metric-bucket-encodings` become a serde alias of relay.metric-bucket-set-encodings?

Reply:
Yes, but it does not matter, Relay will just fall back to the default (which we have everywhere except custom metrics in S4S) until updated.